Tests/Python/PandasConverterTests.cs

```diff
@@ -488,7 +488,7 @@ public void BackwardsCompatibilityDataFrame_get_OnPropertyUsingSymbol()
                 dynamic test = PythonEngine.ModuleFromString("testModule",
                     @"
 def Test(dataFrame, symbol):
-    data = dataFrame.lastprice.get(str(symbol))
+    data = dataFrame.lastprice.get(symbol)
```

Comment:
Maybe its better to do the following to avoid losing any test case for the new and old tests, and it's just 1 more line
```csharp
[TestCase("symbol")]
[TestCase("str(symbol)")]
public void BackwardsCompatibilityDataFrame_get_OnPropertyUsingSymbol(string symbol)
{
    using (Py.GIL())
    {
        dynamic test = PythonEngine.ModuleFromString("testModule",
            $@"
def Test(dataFrame, symbol):
data = dataFrame.lastprice.get({symbol})
if data.empty:
raise Exception('Data is empty')").GetAttr("Test");

        Assert.DoesNotThrow(() => test(GetTestDataFrame(Symbols.SPY), Symbols.SPY));
    }
}